pkg/varlinkapi/util.go

```diff
@@ -133,3 +136,33 @@ func handlePodCall(call iopodman.VarlinkCall, pod *libpod.Pod, ctrErrs map[strin
 
 	return nil
 }
+
+func stringCompressionToArchiveType(s string) archive.Compression {
```

Comment:
Should we document these somewhere in the varlink API docs?

Reply:
I dont think; end user should not need to know about it.

Reply:
> Anything else need to happen to enable the build tests for remote?

which build tests are you talking about?